facades/PC/src/main/java/org/terasology/engine/Terasology.java

```diff
@@ -363,7 +364,12 @@ private static void handleLaunchArguments(String[] args) {
 
         } catch (IOException e) {
             reportException(e);
-            System.exit(0);
+            try {
+                PathManager.getInstance().chooseHomePathManually();
```

Comment:
If this can recover any exception that was thrown before, there is no need to `reportException` before trying the backup. At least, it should not be reported as an error (maybe as a warning, or probably just on INFO level).

Reply:
Yeah , you have a valid point I just thought that it would be good to report an exception without stopping execution. A Warning sounds like a much better approach  :)